frontend/src/metabase/dashboard/actions/tabs.ts

```diff
@@ -0,0 +1,279 @@
+import type { Action } from "redux-actions";
+
+import {
+  DashboardId,
+  DashboardOrderedCard,
+  DashboardOrderedTab,
+  DashboardTabId,
+} from "metabase-types/api";
+import { DashboardState } from "metabase-types/store";
+import { createAction, handleActions } from "metabase/lib/redux";
```

Comment:
Right now I'm still using the `redux-actions` method but in a later PR I can refactor to the redux toolkit equivalents